specs/olm.spec

```diff
@@ -20,6 +20,8 @@ Steps:
   * Verify TektonAddons Install status
   * Validate RBAC
   * Validate default auto prune cronjob in target namespace
+  * Validate tektoninstallersets
```

Comment:
I would rename it to `Validate tektoninstallersets state` so that it's clear why we have two separate steps